cleanlab/internal/validation.py

```diff
@@ -31,13 +31,16 @@ def assert_valid_inputs(
     pred_probs: Optional[np.ndarray] = None,
     multi_label: bool = False,
     allow_missing_classes: bool = True,
+    one_class_allowed: bool = False,
```

Comment:
[nit] should we change this arg name to `allow_one_class` to keep it consistent with `allow_missing_classes`? 
Alternatively, can change `allow_missing_classes` to `missing_classes_allowed`.

Reply:
renamed